hack/cluster-openshift.sh

```diff
@@ -289,7 +307,22 @@ if [[ -f "${OS_ISTIO_OC_EXE_PATH}" ]]; then
   fi
 else
    echo "Downloading binary to ${OS_ISTIO_OC_EXE_PATH}"
-   wget -O ${OS_ISTIO_OC_EXE_PATH} ${OS_ISTIO_OC_DOWNLOAD_LOCATION}
+
+   # Use curl command if available, otherwise try wget
```

Comment:
Minor, but I think this should say
```bash
# Use wget command if available, otherwise try curl
```
Because is trying wget first, if not using curl (no the other way)

Reply:
agreed